nav2_bringup/README.md

```diff
@@ -62,7 +68,9 @@ Notes:
 ```
 
 In RVIZ:
-* Localize the robot using “2D Pose Estimate” button.
+* Add "map" to subscribe topic "/occ_grid"
+* Add "RobotModel", set "Description Source" with "File", set "Description File" with "turtlebot3_burger.urdf(from turtlebot3_description package)"
```

Comment:
This should be generic with a turtlebot3 example:

set "Description FIle" with the name of the urdf file for your robot (example: turtlebot3_burger.urdf)

Reply:
fixed in 98fd017.